index.d.ts

```diff
@@ -0,0 +1,89 @@
+import type { EmitterSubscription } from 'react-native'
```

Comment:
I've merged a PR with TS support - would you mind to rebase this PR on latest main branch to hide these changes? 👀